test/test_jit.py

```diff
@@ -10564,6 +10564,76 @@ def addmm_grad_test(b, x, w):
         self.assertEqual(w.grad, w_ref.grad)
         self.assertEqual(b.grad, b_ref.grad)
 
+    @unittest.skipIf(not RUN_CUDA, "running tests on cuda to verify cudnn fix")
```

Comment:
similar questions to the layernorm:

Should we confirm that batchnorm was indeed auto differentiated?
Also
this test looks very similar to the tests jit_metaprogramming_utils.py should be doing:
fwd result check against ref
grad check against grad_ref
grad grad check against grad grad ref
Is there a reason for a separate test?

Reply:
Good catch, I'm copying the response in the other PR: https://github.com/pytorch/pytorch/pull/50467#discussion_r673040780

Reply:
I have pushed the test to meta programming set.
However, I don't see how to specify test to run on GPU in the meta programming. So I have to kept this test to verify the fix for inference backward in cudnn.